articles/anomaly-detection/passwords-checking.md

```diff
@@ -0,0 +1,229 @@
+Bulk Passwords Checking
+=======================
+
+In Auth0 we focus on bring you the tools you need to keep your systems and your users safe.
+Bulk passwords checking is a tool that allows you to check your user's passwords against pre-defined list of
+passwords to test their strength.
+
+This tool together with our [Breached Password Security](/anomaly-detection/breached-passwords) helps you audit your
+users security and take actions to protect them.
+
+## Bulk Passwords Checking vs Breached Password Security
+Breached Password Security tools checks user passwords against our database of compromised passwords, those passwords usually
```

Comment:
The Breached-Password Detection feature checks user passwords against Auth0's database of compromised passwords. These passwords have been compiled from security breaches experienced by external third-parties, where the breached information have been released to the public. If a compromised account is detected, the Breached-Password Detection tool will take appropriate action.
 +
 +On the other hand, bulk passwords checking aims to give you the power to audit your users security against your own
 +passwords list, those passwords may come from different sources, examples of possible sources might be list of passwords
 +available on the Internet (also known as passwords-dictionaries), your own hand-made checking list, etc.
 +
 +Combined with other tools we offer, actively auditing your users' account passwords will help you to proactively
 +protect them and take risk mitigation actions in case a weak or compromised password gets detected.